### PR TITLE
first version of config parser

### PR DESCRIPTION
**Pull request by RomanPryima**
Please check if cfg_parser is appropriate in general.
For better its featuring we ned to have finished cfg file of our project

### CODE REVIEW

**Comment by Sasha1152** [comment]
Ho is mr. Derek Johnson?

**Comment by RomanPryima** [comment]
It's only a sample of cfg file. 
Right now we have no our own file.

**Comment by UkrainianEngineer** [comment]
It would be great to organize it as a separate function or class, because not it's not useful and we can't easily reuse this.

File: travel/config.cfg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[SectionOne]
```
Comment (UkrainianEngineer) [inline travel/config.cfg:1]: In scope of this task we really don't need any configuration file. It should be added just for test.

File: travel/config_reader.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import ConfigParser
```
Comment (UkrainianEngineer) [inline travel/config_reader.py:4]: As I mentioned, it should be organized as a function or class to make it reusable.

File: travel/config.cfg [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Status: Single
+Name: Derek
+Value: Yes
+Age: 30
+Single: True
+
+[SectionTwo]
+FavoriteColor = Green
```
Comment (UkrainianEngineer) [inline travel/config.cfg:9]: Very often we use "param1=value1" syntax without spaces in configuration files.

**Comment by RomanPryima** [comment]
Remade the module.
Is this correct to commit the result to the branch?

**Comment by UkrainianEngineer** [comment]
Please change target branch from `master` to `release`.

**Review by UkrainianEngineer — COMMENTED** [review]
> Is this correct to commit the result to the branch?

Yes, it's correct.

File: travel/config.cfg (hunk 1 above)
Comment (RomanPryima) [reply travel/config.cfg:1]: Have I to remove testing cfg. file?

File: travel/config.cfg (hunk 3 above)
Comment (RomanPryima) [reply travel/config.cfg:9]: File is a random example from internet. Just for testing.

File: travel/config_reader.py (hunk 2 above)
Comment (RomanPryima) [reply travel/config_reader.py:4]: Does it mean, that import must be inside of function?

**Comment by RomanPryima** [comment]
Closed because of creating new branch. Especially for this task.

**Pull request closed without merging**